frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -391,19 +391,17 @@ def include_attachments(self, message):
 
 @frappe.whitelist()
 def bulk_retry(queues):
-	frappe.only_for("System Manager")
+	if not frappe.has_permission("Email Queue", throw=True):
```

Comment:
should we check for write here? or read check is enough?

(i kind of wanna give people who have read perm on EQ be able to retry)